src/pages/Profile/Profile.jsx

```diff
@@ -1,3 +1,10 @@
+import { useAuth } from '../../hooks';
+import { Text } from '../../components';
+
 export const ProfilePage = () => {
-    return <div>This is profile page</div>;
+    const { user } = useAuth();
+    if (user) {
```

Comment:
this actually should be handled in ProtectedRoute

Reply:
Yup it is just an example usage. It will be removed before requesting a review. It is just for my testing purposes.